Cargo.toml

```diff
@@ -19,7 +19,7 @@ vendored-openssl = ["libssh2-sys/vendored-openssl"]
 [dependencies]
 bitflags = "1.2"
 libc = "0.2"
-libssh2-sys = { path = "libssh2-sys", version = "0.2.16" }
+libssh2-sys = { path = "libssh2-sys", version = "0.3.0" }
```

Comment:
The -sys changes can be a simple bump; they add to the underlying `-sys` API but don't change any existing part of it.  This is important because parts of cargo depend on the sys crate directly.

`version = "0.2.17"`

Reply:
> but don't change any existing part of it

Actually it does change the type of existing constants from `c_int` to `c_ulong`.

Reply:
Please change them back to c_int so that we can make the next release `0.2.17` and not change the interface of the sys crate in an incompatible way!

Reply:
I'm not convinced they should remain as `c_int`. Actually, the C header of `libssh2` should have all those `LIBSSH2_FX_*` constants suffixed by `UL` to make sure they are typed as `unsigned long` and not simply as the default `int`.

In the meantime, I'll convert them back to `c_int`, so that your workflow proceeds...